ushahidi/platform-client
# 1000 image upload

This pull request makes the following changes:
- Add image field to post type

Test these changes by:
- Add an image field to a post type
- Create a new post with the post type and upload an image
- The image should be uploaded and referenced as a post value.
- The image should be shown in the post list as well as post detail views

Fixes ushahidi/platform#1000 .

Ping @ushahidi/platform


---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/167)
